.gitignore

```diff
@@ -1,4 +1,4 @@
-files/
+media/
```

Comment:
consider adding a comment that mentions this is equal to MEDIA_ROOT in django's settings

Reply:
deferred for #7